cli/popper/commands/cmd_run.py

```diff
@@ -42,6 +49,8 @@ def cli(ctx, pipeline, timeout, skip):
         sys.exit(0)
 
     if pipeline:
+        if ignore_errors:
+            pu.info("W: ignore-errors flag is ignored when pipeline argument is provided")
```

Comment:
does the `W` stand for WARNING? If that's the case, then we could add a new utility function named `warn`